src/Sentry/IHub.cs

```diff
@@ -17,6 +17,20 @@ public interface IHub : ISentryClient, ISentryScopeManager
     /// </summary>
     public SentryId LastEventId { get; }
 
+    /// <summary>
+    /// Creates and sends logs to Sentry.
+    /// <para>This API is experimental and it may change in the future.</para>
+    /// </summary>
+    /// <remarks>
+    /// Available options:
+    /// <list type="bullet">
+    /// <item><see cref="Sentry.SentryOptions.EnableLogs"/></item>
+    /// <item><see cref="Sentry.SentryOptions.SetBeforeSendLog(System.Func{SentryLog, SentryLog})"/></item>
+    /// </list>
+    /// </remarks>
+    [Experimental(Infrastructure.DiagnosticId.ExperimentalFeature)]
+    public SentryStructuredLogger Logger { get; }
```

Comment:
Just thinking about testing of the Hub classes in general, the `Logger` is kind of a dependency of the hub and currently it's not very easy to inject/mock.

Maybe worth creating an `ISentryStructuredLogger` interface and having this member be of type `ISentryStructuredLogger` rather than `SentryStructuredLogger`. A mock Logger could then optionally be injected via the constructor for test purposes.

Reply:
I am a bit hesitant about an interface ... since adding to it is breaking.
Well ... to be fair ... we offer no facility for the user to inject a custom `SentryStructuredLogger` type, so implementing that interface is not a scenario.

I did make the `SentryStructuredLogger` now abstract, with a `Default..` and a `Disabled..` implementation, where we could later add a `Test..` implementation in `Sentry.Testing` which logs to an in-memory-collection and offers assertions/verifications.